kv/org.go

```diff
@@ -267,6 +267,10 @@ func (s *Service) CreateOrganization(ctx context.Context, o *influxdb.Organizati
 			s.Logger.Info("failed to make user owner of organization", zap.Error(err))
 		}
 
+		if err := s.createSystemBuckets(ctx, tx, o); err != nil {
```

Comment:
Should we fail here? Users won't be able to create tasks if their system buckets don't exist.

Reply:
good catch! this change pointed out another issue where we validate bucket name.